framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
At present state this controller is empty: in such state it does not make any sense.
Usually queues are processed by deamons - not console commands. Still it could be implemented by console command running by cron with mutex.
Either way - it is better to be decided by developer.